ld_trials.js

```diff
@@ -0,0 +1,177 @@
+/*
+ * This file contains all the possible discrete sub trials of
+ * a lexical decision task.
+ */
+
+/*
+ * Presents the fixation cross
+ */
+let present_fixation = {
+    type: jsPsychHtmlKeyboardResponse,
+    stimulus: '<span style="font-size:40px;">+</span>',
+    choices: "NO_KEYS",
+    trial_duration: FIXCROSS_DURATION
+};
+
+/*
+ * This stimulus will present a forward mask
+ */
+let forward_mask = {
+    type: jsPsychHtmlKeyboardResponse,
+    stimulus: function(){
+        return "<p class='stimulus'>" +
+            jsPsych.timelineVariable('forward_mask', true) +
+            "</p>";
+    },
+    choices: "NO_KEYS",
+    trial_duration: MASK_DURATION,
+    data: { useful_data_flag: false }
+};
+
+/*
+ * This stimulus will present a visual prime stimulus
+ */
+let visual_prime = {
+    type: jsPsychHtmlKeyboardResponse,
+    stimulus: function(){
+        return "<p class='stimulus'>" +
+            jsPsych.timelineVariable('visual_prime', true) + "</p>";
+    },
+    choices: "NO_KEYS",
+    trial_duration: PRIME_DURATION,
+    post_trial_gap: PRIME_GAP_DURATION
+};
+
+/*
+ * This will present an auditory prime stimulus.
+ */
+let auditory_prime = {
+    type : jsPsychAudioKeyboardResponse,
+    stimulus : function() { return jsPsych.timelineVariable("auditory_prime");},
```

Comment:
the extra function wrapper isn't required in that case

Reply:
Ah, this was your point:-)

Reply:
Actually, now i'm putting it back, and I see why I put it there in the first place.
As it gives me: `Uncaught ReferenceError: jsPsych is not defined`
I've put it in a function, because at runtime.
```javascript
let jsPsych = let jsPsych = initJsPsych(...);
```
has run, however, when the browser is parsing  ld_trials.js, it's not and I get the error above.
I think we can fix this more properly when we adopt a modern module in the scripts as well.